charts/generic/node-cpu-hog/experiment.yaml

```diff
@@ -55,6 +55,15 @@ spec:
     - name: RAMP_TIME
       value: ''
 
+    ## ENTER THE NUMBER OF CORES OF CPU FOR CPU HOGGING
+    ## OPTIONAL VALUE IN CASE OF EMPTY VALUE IT WILL TAKE NODE CPU CAPACITY 
+    - name: NODE_CPU_CORE
+      value: ''
+
```

Comment:
Lets add a LIB as well and place litmus there, though that is the only one we support at this point.